espanol/canudos.md

```diff
@@ -41,6 +41,8 @@ and doesn't know how to speak Spanish.](../english/TheEpicSax/111_TheEpicSaxGuyA
 
 [Subió a la torre](torre/torre.md)
 
+[Se cansó de la guerra y decidió alienarse de todo lo material, accediendo a un estado de meditación transcendente](transcender/transcender.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.